src/main/java/uk/gov/hmcts/reform/sscs/builder/TrackYourAppealJsonBuilder.java

```diff
@@ -283,6 +274,10 @@ private ArrayNode buildEventArray(List<Event> events, Map<Event, Document> event
         return latestEvents;
     }
 
+    private boolean isCaseStateReadyToList(SscsCaseData caseData) {
+        return State.READY_TO_LIST.toString().equals(caseData.getCreatedInGapsFrom()) ? true : false;
```

Comment:
Won't old cases not have this field so will throw a nullpointer here? I can see other code in tribunals checking for null first

Reply:
State.READY_TO_LIST is static string and here it will not throw null pointer exception if field is caseData.getCreatedInGapsFrom() null